src/test/objectstore/Allocator_test.cc

```diff
@@ -341,12 +341,14 @@ TEST_P(AllocTest, test_dump_fragmentation_score)
 	//allocate
 	want_size = ( rng() % one_alloc_max ) / alloc_unit * alloc_unit + alloc_unit;
 	tmp.clear();
-	uint64_t r = alloc->allocate(want_size, alloc_unit, 0, 0, &tmp);
+	int64_t r = alloc->allocate(want_size, alloc_unit, 0, 0, &tmp);
+  if (r > 0) {
```

Comment:
could you please indent the whole `if` block accordingly?

Reply:
> could you please indent the whole `if` block accordingly?

ok, i will. thanks